src/react/workflow/WorkflowList.jsx

```diff
@@ -8,8 +8,13 @@ import { ListSection } from '../ui-elements/ListLayout3';
 import React from 'react';
 import type { Rules } from '../../types/config';
 import { push } from 'connected-react-router';
+import styled from 'styled-components';
 import { useDispatch } from 'react-redux';
 
+const WorkflowTypeCell = styled.span`
```

Comment:
We might want to create a `ui-elements/Utility.jsx` file and import it from there for reusability.

Reply:
done